actions.go

```diff
@@ -25,6 +25,11 @@ import (
 	"strings"
 )
 
+var (
+	// osExit allows `os.Exit()` to be stubbed during testing.
+	osExit = os.Exit
```

Comment:
This change is "necessary" to get test coverage for the `os.Exit()` call in `Fatalf()`.

However, the "uncertainty" of a var here may not be worth it to you, so I am happy to revert this change.

Going in the **OTHER** direction, we could also put an `ExitFunc` on `Action` the same way we did for `GetenvFunc`.